cyences_app_for_splunk/default/data/ui/views/cs_kaspersky_reports.xml

```diff
@@ -198,4 +198,31 @@
       </table>
     </panel>
   </row>
+  <row>
+    <panel>
+      <title>Kaspersky Critical Host</title>
```

Comment:
May be require to change the panel title, its not very clear what it is looking for.

Reply:
Shall we create alert for this as well?? How critical is this??